src/auth.rs

```diff
@@ -271,7 +271,7 @@ impl<'l> FromRequest<'l> for InvokeAuthWrapper {
                 };
                 match target.service() {
                     None => bad_request(anyhow!("missing service in invocation target")),
-                    Some("s3") => {
+                    Some("kv") => {
```

Comment:
Shall I make an issue to turn those into enums?

Reply:
yeah we'll probably want that in the long run